specs/codecs/wac/spec.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "WAC Specification"
+navTitle: "Spec"
+---
+
+# Specification: WAC
+
+**Status: Prescriptive - Exploratory **
+
+* [Format](#format)
+* [Implementations](#implementations)
+    * [Go](#go)
+    * [Rust/WASM](#rustwasm)
```

Comment:
What's the right way to do this? `error: bad link specs/codecs/wac/spec#rustwasm at line 14`

I could just change the title to `Rust and WASM`, but also curious what the answer is

Reply:
it wants `#rust-wasm` (confirmed by looking at the built file: `<h3 id="rust-wasm" tabindex="-1">`)